index.html

```diff
@@ -23,25 +23,162 @@
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
+    </section>
+
+    <section id="tonic" class="card">
+
+        <img src="images/portfolio3.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Tonic</h1>
+        <img src="images/fame2.png" alt="frame" id="frame">
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
+    </section>
+
+    <section id="multi2" class="card">
+
+        <img src="images/portfolio4.png" alt="a_screentshot" id="portfolio">
+
+        <h1 class="title tonic">Multi-Post Stories</h1>
+
+        <img src="images/fame2.png" alt="frame" id="frame">
+
+        <p class="tonic-p">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+        
+        <ul class="code-links">
+            <li class="tag">
+                <a href="" alt="">HTML</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">CSS</a>
+            </li>
+            <li class="tag">
+                <a href="" alt="">JavaScript</a>
+            </li>
+        </ul>
+
+        <h3 class="button"><a href="" id="linkp">See Project</a></h3>
+
+    </section>
+
+    <section id="aboutme">
+
+        <h1 class="title">
+                About Myself
+            </h1>
+            <p class="text">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
+            <h3>LET’S CONNECT</h3>
+
+            <ul class="social">
+                <li><a href="#"><img src="images/twitter.svg" alt="twitter_logo" id="twitter" class="icon"></a></li>
+                <li><a href="#"><img src="images/linkedin.svg" alt="linkedin_logo" id="linkedin" class="icon"></a></li>
+                 <li><a href="#"><img src="images/hand.svg" alt="hand" id="hand" class="icon"></a></li>
+                <li><a href="#"><img src="images/git.svg" alt="github_logo" id="github" class="icon"></a></li>
+                <li><a href="#"><img src="images/M.svg" alt="capital_m" id="m" class="icon"></a></li>
+            </ul>
+
+            <h3 class="button" id="get"><a href="" id="">Get my resume</a></h3>
+
+            <img src="images/skills.png" alt="skills_list" id="skills">
+   
```

Comment:
- [ ]  Please make sure that the technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`. Note that the technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li`. Currently, you have inserted an image representing these, and remember that this is a bad practice :smile: